src/qt/bitcoingui.cpp

```diff
@@ -672,7 +672,7 @@ void BitcoinGUI::setNumConnections(int count)
     labelConnectionsIcon->setToolTip(tr("%n active connection(s) to Bitcoin network", "", count));
 }
 
-void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate)
+void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate, const CBlockIndex *tip)
```

Comment:
Hmm okay you have a point.
But ideally these kind of things happen in the model (ClientModel) in this case, not in the view class.
Don't like how core's handles leak all over the place now.
But yes maybe better to live with this for now...